### PR TITLE
fix: add ayushlohmod

### LINKED ISSUE #1229: Ayush Lohmod
## Profile Screenshot

Screenshot 2022-07-13 at 8 29 02 PM

### PR DESCRIPTION
**Pull request by ayushlohmod**
resolves #1229

### CODE REVIEW

File: profiles/ayushlohmod.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/37297939/178766012-d773a89b-8627-4ca7-bdad-5def06995121.png",
+    "issueId": 1229,
+    "name": "Ayush Lohmod",
+    "username": "ayushlohmod"
+  }
```
Comment (Panquesito7) [inline profiles/ayushlohmod.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/ayushlohmod.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+"image": "https://user-images.githubusercontent.com/37297939/178766012-d773a89b-8627-4ca7-bdad-5def06995121.png",
+"issueId": 1229,
+"name": "Ayush Lohmod",
+"username": "ayushlohmod"
```
Comment (Panquesito7) [inline profiles/ayushlohmod.json:5]: It's missing a **two-space** indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/37297939/178766012-d773a89b-8627-4ca7-bdad-5def06995121.png",
  "issueId": 1229,
  "name": "Ayush Lohmod",
  "username": "ayushlohmod"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by ayushlohmod — COMMENTED** [review]
please have a look on this

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient, Elegant

**Pull request merged**